app/assets/stylesheets/styles/game.scss

```diff
@@ -1,7 +1,9 @@
 #gamesheader {
-  color: #9bb0bb;
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.